src/webportal/src/app/user/fabric/batchRegister/Table.jsx

```diff
@@ -0,0 +1,361 @@
+          <Dropdown
+            multiSelect
+            options={options}
+            defaultSelectedKeys={vcs}
+            onChange={(_event, option, _index) => {
+              if (option.selected) {
+                vcs.push(option.text);
+              } else {
+                vcs.splice(vcs.indexOf(option.text), 1);
+              }
+              userInfo['virtual cluster'] = vcs.join(',');
+            }}
+            onRenderTitle={(_options) => {
+              return (
+                <div>
+                  {getDisplayVirtualClusterString(userInfo['virtual cluster'])}
+                </div>
+              );
+            }}
+          />
+        );
+      }
+    },
+  };
+
+  /**
+   * @type {import('office-ui-fabric-react').IColumn}
+   */
+  const githubPATColumn = {
+    key: 'githubPAT',
+    minWidth: 80,
+    maxWidth: 200,
+    name: 'Github PAT',
+    className: FontClassNames.mediumPlus,
+    headerClassName: FontClassNames.medium,
+    isResizable: true,
+    onRender: (userInfo) => {
+      const {githubPAT} = userInfo;
+      return (
+        <TextField
+          readOnly={isFinished(userInfo)}
+          defaultValue={githubPAT}
+          onChange={(_event, newValue) => {
+            userInfo.githubPAT = newValue;
+          }}
+        />
+      );
+    },
+  };
+  githubPATColumn.onRender.displayName = 'onRenderGithubPATColumn';
+
+  /**
+   * @type {import('office-ui-fabric-react').IColumn}
+   */
+  const statusColumn = {
+    key: 'status',
+    minWidth: 100,
+    maxWidth: 100,
+    name: 'Status',
+    className: FontClassNames.mediumPlus,
+    headerClassName: FontClassNames.medium,
+    isResizable: true,
+
+    onRender(userInfo) {
+      const {status: {isSuccess, message} = {}} = userInfo;
+
+      /** @type {React.CSSProperties} */
+      const wrapperStyle = {display: 'inline-block', verticalAlign: 'middle', width: '100%'};
+      let statusText = undefined;
+      if (isSuccess === true) {
+        statusText = 'Succeeded';
+      } else if (isSuccess === false) {
+        statusText = 'Failed';
+      } else {
+        return undefined;
+      }
+      /** @type {MessageBarType} */
+      const messageBarType = {
+        Succeeded: MessageBarType.success,
+        Failed: MessageBarType.remove,
+      }[statusText];
+      const rootStyle = {
+        backgroundColor: {
+          Succeeded: '#7FBA00',
+          Failed: '#E81123',
+        }[statusText],
+      };
+      /** @type {import('@uifabric/styling').IStyle} */
+      const iconContainerStyle = {marginTop: 8, marginBottom: 8, marginLeft: 8};
+      /** @type {import('@uifabric/styling').IStyle} */
+      const iconStyle = {color: 'white'};
+      /** @type {import('@uifabric/styling').IStyle} */
+      const textStyle = {marginTop: 8, marginRight: 8, marginBottom: 8, color: 'white'};
+      return (
+        <div style={Object.assign(wrapperStyle)}>
+          <TooltipHost content={message}>
+            <MessageBar
```

Comment:
Please use the statusBadge component in src/app/components. (You may need rebase/merge your branch to master...)

Reply:
Rebased and used statusBadge, thanks for your suggestion.